### PR TITLE
Greps for double firelocks and airlocks

### PR DESCRIPTION
**Pull request by Son-of-Space**
## About The Pull Request

Adds the greps for firelocks and airlocks so we don't double stack them.

## Why It's Good For The Game
consistency, catching errors before they go on live, all the fun stuff.

## Changelog

:cl: Son-of-Space, Pepsilawn
code: Adds checks for double firelocks or airlocks
fix: fixes several double firelocks and airlocks across our maps
/:cl:

### CODE REVIEW

**Comment by Son-of-Space** [comment]
I don't think my regex here is working properly given I know there's at least one double firelock somewhere. Could someone who has a bit more knowledge on regex assist? Thank you. : )

File: tools/ci/check_grep.sh [hunk 1]
```diff
@@ -69,6 +69,16 @@ if grep -Pzo '"\w+" = \(\n[^)]*?/obj/structure/chair/(?<type>[/\w]*),\n[^)]*?/ob
     echo "ERROR: found multiple identical chairs on the same tile, please remove them."
     st=1
 fi;
+if grep -Pzo '"\w+" = \(\n[^)]*?/obj/machinery/door/airlock/(?<type>[/\w]*),\n[^)]*?/obj/machinery/door/airlock/\g{type},\n[^)]*?/area/.+\)' _maps/**/*.dmm;	then
```
Comment (Pepsilawn) [inline tools/ci/check_grep.sh:72]:
```suggestion
if grep -Pzo '"\w+" = \(\n[^)]*?/obj/machinery/door/airlock[/\w]*?,\n[^)]*?/obj/machinery/door/airlock[/\w]*?,\n[^)]*?/area/.+\)' _maps/**/*.dmm;	then
```
Haven't touched Perlre in forever so beware but it should help detect more valid cases where different subtypes are still on the same tile. Potentially turn both grep checks into a single one for /obj/machinery/door/ as I can't think of an instance where we want both windoors and airlocks on the same tile, or any other subtypes of it.

File: tools/ci/check_grep.sh [hunk 2]
```diff
@@ -69,6 +69,16 @@ if grep -Pzo '"\w+" = \(\n[^)]*?/obj/structure/chair/(?<type>[/\w]*),\n[^)]*?/ob
...
+	echo
+    echo "ERROR: found multiple airlocks on the same tile, please remove them."
+    st=1
+fi;
+if grep -Pzo '"\w+" = \(\n[^)]*?/obj/machinery/door/firedoor/[^/](?<type>[/\w]*),\n[^/)]*?/obj/machinery/door/firedoor/[^/]\g{type},\n[^)]*?/area/.+\)' _maps/**/*.dmm;	then
```
Comment (Pepsilawn) [inline tools/ci/check_grep.sh:77]:
```suggestion
if grep -Pzo '"\w+" = \(\n[^)]*?/obj/machinery/door/firedoor[/\w]*?,\n[^)]*?/obj/machinery/door/firedoor[/\w]*?,\n[^)]*?/area/.+\)' _maps/**/*.dmm;	then
```
Same as above

**Comment by Son-of-Space** [comment]
I think the code works here so this is ready for review

**Pull request merged**